src/ens.js

```diff
@@ -79,77 +78,59 @@ export class ENS {
 
   /* Main methods */
 
+  // TODO: ethers.js does not support owner
   async getOwner(name) {
     const namehash = getNamehash(name)
     const owner = await this.ENS.owner(namehash)
     return owner
   }
 
   async getResolver(name) {
```

Comment:
This should probably be called `getResolverAddress`, and `getResolverObject` should be renamed to `getResolver`. This function can then call that one and just unwrap the resulting object.

Reply:
`getResolver` is a public function so I don't want to change the interface and also shouldn't return the entire ethersjs's resolver object unless necessary (our library interface changes whenever ethers.js interface changes). I can  rename `getResolverObject` to `_getResolverObject` to make it look more like private function if necessarily.